mos_tests/neutron/python_tests/base.py

```diff
@@ -140,3 +140,49 @@ def check_vm_is_connectable(self, vm):
             wait(lambda: remote.execute(cmd)['exit_code'] == 0,
                  interval=10, timeout=3 * 10,
                  timeout_msg=error_msg)
+
+    def destroy_nodes(self, devops_nodes):
```

Comment:
I believe you can separate destructive action and check action. And move them to devops client and fuel environment.